app/views/runs/_form.html.erb

```diff
@@ -22,18 +22,44 @@
     <div class="field">
       <% if runner_role == 'Coach' %>
       <%= f.label :training_plan %><br>
-      <%= f.text_area :training_plan, rows:"6" %>
+      <%= f.text_area :training_plan, rows:"6", class: 'form-control' %>
       <% end %>
     </div>
     <div class="field">
       <%= f.label :progress %><br>
-      <%= f.text_area :progress, rows:"6" %>
+      <%= f.text_area :progress, rows:"6", class: 'form-control' %>
     </div>
 
+    <div class="field">
+      <% if runner_role == 'Coach' %>
+      <%= f.label :changed_flag, 'Custom' %>
+      <%= f.check_box :changed_flag %>
+      <% end %>
+    </div>
 
     <div class="actions">
-      <%= f.submit %>
+      <%= f.submit 'Update Run', :class => 'btn btn-info' %>
     </div>
 
 
-<% end %>
\ No newline at end of file
+<% end %>
+
+<script>
+  //Javascrip to test if training_plan is different
+  var training_plan;
+
+  $(document).ready(function(){
+      training_plan = $('#run_training_plan').val();
+  });
+
+    $('#run_training_plan').change(function(){
```

Comment:
Humm...   I think this is interesting, but I am not sure about this logic being in Javascript, versus being a method on the Model object....    @omnifroodle or @ychaker I'd love any thoughts on this.